src/main/resources/changesets/execution_backend_info.xml

```diff
@@ -0,0 +1,87 @@
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="add_foreign_key_constraints" author="mcovarr">
+        <addForeignKeyConstraint constraintName="FK_EXECUTION_INFO_EXECUTION"
+                                 baseTableName="EXECUTION_INFO" baseColumnNames="EXECUTION_ID"
+                                 referencedTableName="EXECUTION"
+                                 referencedColumnNames="EXECUTION_ID"/>
+    </changeSet>
+
+    <changeSet id="add_backend_type_column" author="mcovarr">
+        <addColumn tableName="EXECUTION">
+            <column name="BACKEND_TYPE" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="insert_data_from_backend_tables" author="mcovarr">
+        <sql>
+            insert into EXECUTION_INFO(EXECUTION_ID, INFO_KEY, INFO_VALUE)
+            select EXECUTION_ID, "JES_RUN_ID", JES_RUN_ID from JES_JOB;
+
+            insert into EXECUTION_INFO(EXECUTION_ID, INFO_KEY, INFO_VALUE)
+            select EXECUTION_ID, "JES_STATUS", JES_STATUS from JES_JOB;
+
+            insert into EXECUTION_INFO(EXECUTION_ID, INFO_KEY, INFO_VALUE)
+            select EXECUTION_ID, "PID", PID from LOCAL_JOB;
+
+            insert into EXECUTION_INFO(EXECUTION_ID, INFO_KEY, INFO_VALUE)
+            select EXECUTION_ID, "SGE_JOB_NUMBER", SGE_JOB_NUMBER from SGE_JOB;
+        </sql>
+    </changeSet>
+
+    <changeSet id="set_backend_type" author="mcovarr">
+        <sql>
+            update EXECUTION e set BACKEND_TYPE = "JES" where exists
+            (select 1 from JES_JOB jj where jj.EXECUTION_ID = e.EXECUTION_ID);
+
+            update EXECUTION e set BACKEND_TYPE = "SGE" where exists
+            (select 1 from SGE_JOB sj where sj.EXECUTION_ID = e.EXECUTION_ID);
+
+            update EXECUTION e set BACKEND_TYPE = "Local" where exists
+            (select 1 from LOCAL_JOB lj where lj.EXECUTION_ID = e.EXECUTION_ID);
+        </sql>
+    </changeSet>
+
+    <changeSet id="drop_job_tables" author="mcovarr">
+        <dropTable tableName="jes_job" schemaName="cromwell"/>
+        <dropTable tableName="sge_job" schemaName="cromwell"/>
+        <dropTable tableName="local_job" schemaName="cromwell"/>
+    </changeSet>
+
+    <changeSet id="execution_info_uniqueness_constraint" author="mcovarr">
+        <addUniqueConstraint constraintName="UK_EXECUTION_INFO"
+                             tableName="EXECUTION_INFO"
+                             schemaName="CROMWELL"
+                             columnNames="EXECUTION_ID, INFO_KEY" />
+    </changeSet>
+
+    <changeSet id="execution_backend_type_not_null" author="mcovarr">
+        <addNotNullConstraint tableName="EXECUTION"
+                              columnName="BACKEND_TYPE"
+                              schemaName="CROMWELL"
+                              columnDataType="VARCHAR(255)"/>
+    </changeSet>
+
```

Comment:
Metadata will need to be richer than just those.  Cromwell will want to record the backend type used for a call, but that is treated as just an opaque string.